.devops/code-review-pipelines.yml

```
@@ -11,7 +11,7 @@ resources:
     - repository: pagopaEcommerceLocal
       type: github
       name: pagopa/pagopa-ecommerce-local
-      ref: CHK-4172-add-commands-integration-tests
+      ref: CHK-4267-inject-api-key-into-tests
```

```suggestion
      ref: main
```

TODO: to be replace after dependent PR merge